integrationTests/chainSimulator/staking/jail_test.go

```diff
@@ -152,9 +152,121 @@ func testChainSimulatorJailAndUnJail(t *testing.T, targetEpoch int32, nodeStatus
 	checkValidatorStatus(t, cs, blsKeys[0], "eligible")
 }
 
+// Test description
+// Add a new node and wait until the node get jailed
+// Add a second node to take the place of the jailed node
+// UnJail the first node --> should go in queue
+// Activate staking v4 step 1 --> node should be moved from queue to auction list
```

Comment:
missing test comment with the scenario number

Reply:
added